ui/package.json

```diff
@@ -27,14 +27,12 @@
     "start": "VAULT_ADDR=http://127.0.0.1:8200; ember server --proxy=$VAULT_ADDR",
     "start2": "ember server --proxy=http://127.0.0.1:8202 --port=4202",
     "start:chroot": "ember server --proxy=http://127.0.0.1:8300 --port=4300",
-    "start:mirage": "start () { MIRAGE_DEV_HANDLER=$1 yarn run start; }; start",
```

Comment:
in yarn 3.x, you can no longer define and invoke inline functions. I spent _way_ more time researching this than I'd like to admit. 

The only alternatives I found were to add a new dependency ([cross-env](https://www.npmjs.com/package/cross-env) - which is way more robust than we need) or writing a separate script file. `export MIRAGE_DEV_HANDLER=<handler> yarn start` is pretty simple and didn't feel necessary to over complicate by either of the aforementioned solutions